### ISSUE TITLE
Write firmware to incorporate feedback from drive encoders

### ISSUE THREAD

**Issue by rpshukla** [UofA-SPEAR/embedded#55]

**Comment by davidlenfesty** [comment]
This is blocked on motor/encoder selection, likely won't happen until late in this semester.

**Comment by JackOpgenorth** [comment]
This issue will be on hold for a bit
